### ISSUE TITLE
'tv' vs 'television' for category names

### ISSUE THREAD

**Issue by michaelhsmart** [jskinn/rvchallenge-starter-kit#1]
This is barely a bug, but I found when applying a coco detector to this challenge that your class list uses 'television' while the COCO category names use 'tv'. It's pretty easy to handle on our end, but I figured I'd open an issue for this in case you wanted them to exactly match with COCO.

Thanks for hosting this challenge!

**Comment by david2611** [comment]
Thanks for participating :+1: 

If the "classes" parameter in your submission.json contains "tv" our evaluation code should be able to handle this just fine.

We have a list of synonyms that we should be able to handle in the class_list.py file of our evaluation code (https://github.com/jskinn/rvchallenge-evaluation/blob/master/class_list.py). These get interpreted as the appropriate classes in our evaluation process.

Have you tried a submission where your class list included "tv"?

**Comment by michaelhsmart** [comment]
Not yet. I was working on our conversion map to fine-tune train on the RV validation set and stumbled on to this difference via a KeyError.

On that note though, if you have that sort of system already integrated with the ACRV challenge, would you happen to also have any recommended/supported COCO <-> ACRV format converters already existing? I don't see any in either of the challenge repositories. Just want to check before I write my own.

**Comment by david2611** [comment]
No worries. We currently have a method for converting COCO detection outputs into our format (referred to in the code as rvc1 format) available in our related pdq_evaluation package that was recently released (https://github.com/david2611/pdq_evaluation). The program you need is file_convert_coco_to_rvc1.py. 

This doesn't change the class names but might help if you have a system trained to give COCO outputs. The evaluation code should be able to take it from there.

The package also comes with some extra visualisation tools which may help with debugging.

This doesn't however have an easy converter back to COCO format detection files but as a starting point you should be able to use convert_to_coco_input.py in that pdq_evaluation package.
